README.md

```diff
@@ -1 +1,90 @@
-# copa-action
\ No newline at end of file
+# Copa Action
+
+This action patches vulnerable containers using [Copa](https://github.com/project-copacetic/copacetic).
+
+## Inputs
+
+## `image`
+
+**Required** The image reference to patch.
+
+## `image-report`
+
+**Required** The trivy json vulnerability report of the image to patch.
+
+## `patched-tag`
+
+**Required** The patched image tag to append to the original tag.
+
+## Output
+
+## `patched-image`
+
+Image reference of the resulting patched image.
+
+## Example usage
+
+```
+on: [push]
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+
+        strategy:
+          fail-fast: false
+          matrix:
+            # provide relevant list of images to scan on each run
+            images: ['docker.io/library/nginx:1.21.6', 'docker.io/openpolicyagent/opa:0.46.0', 'docker.io/library/hello-world:latest']
+
+        steps:
+        - name: Checkout repository
```

Comment:
is this a requirement? sounds like we need this for Dockerfile? can we prebuilt these or bake these into the action?

Reply:
https://github.com/aquasecurity/trivy-action is similar but doesn't require checkout of the repo for example

Reply:
the trivy action is using docker instead of a composite action, so we might have to prebuild and push the image. I can create a separate PR with just the Dockerfile?

Reply:
what's difference between docker vs composite?

Reply:
instead of one docker container action it can have multiple steps in the action, this allowed us to to give copa access to buildkit